docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -129,7 +129,7 @@ of the set of pods. A cron job does not examine pods at all.
 
 As with all other Kubernetes configs, a cron job needs `apiVersion`, `kind`, and `metadata` fields. For general
 information about working with config files, see [deploying applications](/docs/user-guide/deploying-applications),
-[configuring containers](/docs/user-guide/configuring-containers), and
+[configuring containers](/docs/tasks/configure-pod-container/assign-memory-resource/), and
```

Comment:
I don't think the new link is good.

Reply:
In the docs directory structure：

```shell 
$ ls docs/user-guide/user-guide/configuring*
ls: cannot access docs/user-guide/user-guide/configuring*: No such file or directory
```

Reply:
I wasn't saying that the old link should be kept. Either we find a better, meaningful target for the link, or we can delete the link if it is not that relevant any more. Pointing that to memory resource assignment makes no sense to me.

Reply:
Thanks，restore it.